pyannote/audio/interactive/v03.py

```diff
@@ -0,0 +1,193 @@
+from copy import deepcopy
+from pathlib import Path
+from typing import Any, Dict, Iterable, List, Optional, Union
+
+import prodigy
+from answers2file import answers2file
+from prodigy.components.db import connect
+from prodigy.components.loaders import Audio
+from prodigy.core import recipe
+from prodigy.util import log
```

Comment:
Isn't it available as `prodigy.log` already?  and `prodigy.recipe`? Would be more readable I think.